api/collections/urls.py

```diff
@@ -0,0 +1,15 @@
+from django.conf.urls import url
+from api.collections import views
+
+urlpatterns = [
+    # Examples:
+    # url(r'^$', 'api.views.home', name='home'),
+    # url(r'^blog/', include('blog.urls')),
+    url(r'^$', views.CollectionList.as_view(), name='collection-list'),
+    url(r'^dashboard/$', views.DashboardDetail.as_view(), name='dashboard-detail'),
+    url(r'^[~]?(?P<pk>\w+)/$', views.CollectionDetail.as_view(), name='collection-detail'),
```

Comment:
We shouldn't be using <pk> anywhere anymore. We should be using <collection_id> instead. See #3121 for the most specific changes you'll need to make to urls, serializers, and views.